src/containers/Ledger/test/Ledger.test.js

```diff
@@ -64,11 +72,14 @@ describe('Ledger container', () => {
 
     expect(summary.length).toBe(1)
     expect(summary.find('.ledger-cols').length).toBe(1)
-    expect(summary.find('.ledger-col').length).toBe(3)
-    expect(summary.find('.ledger-index').length).toBe(1)
     expect(summary.find('.closed-date').length).toBe(1)
     expect(summary.find('.ledger-hash').length).toBe(1)
 
+    // this is calculated for this specific ledger data
+    // 4 general cols (# of txns, # of successful txns, # of failed txns)
+    // 6 transaction cols (# of account txns, # of dex txns, # of nft txns, # of payment txns, # of xchain txs, # of pseudo txns)
+    expect(summary.find('.ledger-col').length).toBe(10)
```

Comment:
There should be a test that  checks the values of each of the tx types are correct and not just the number of elements.